project.clj

```diff
@@ -34,16 +34,9 @@
                  ;; 'uberjar-exclusions' example toward the end of this file.
                  [org.jruby/jruby-stdlib "1.7.26"]
 
-
-                 [org.clojure/tools.logging "0.3.1"]
```

Comment:
Looks like we'd get version `0.2.6` from ![clj-parent](https://github.com/puppetlabs/clj-parent/blob/0.1.3/project.clj#L14) -- which is a lower version than what we would have had previously.  Not sure if the newer version was intentional / required here?